docs/source/guides/user/chapters/installing.rst

```diff
@@ -119,13 +119,25 @@ Installing from source code
 First make sure you have a basic set of packages installed. The following
 applies to Fedora based distributions, please adapt to your platform::
 
-    $ sudo dnf install -y python3 git gcc python3-devel python3-pip libvirt-devel libffi-devel openssl-devel libyaml-devel redhat-rpm-config xz-devel
+    $ sudo dnf install -y python3 git gcc python3-pip
 
 Then to install Avocado from the git repository run::
 
     $ git clone git://github.com/avocado-framework/avocado.git
     $ cd avocado
-    $ sudo python3 setup.py install
+    $ python3 setup.py install --user
+
+Optionally, to install the plugins run:
+
+    $ python3 setup.py plugin --install=golang --user
```

Comment:
I think we should have a generated page per plugin on our docs, with an index and refer to them here.  The idea is to avoid a maintenance burden and having this list out of sync with the existing plugins.  May be done in a separate PR.

Reply:
> I think we should have a generated page per plugin on our docs, with an index and refer to them here. The idea is to avoid a maintenance burden and having this list out of sync with the existing plugins. May be done in a separate PR.

Ok, do you mind filing a bug with the idea you have in mind?

Reply:
Another idea is to have each plugin with its own README.rst updated (since some people might reach a plugin by the github page directly) and try to reuse this README inside our docs somehow without content duplication, so when a plugin'sREADME is updated our documentation will be also updated.